plugins/modules/apk.py

```diff
@@ -371,6 +371,12 @@ def main():
     if p['upgrade']:
         upgrade_packages(module, p['available'])
 
+    if all(not name.strip() for name in p['name']):
+        if len(p['name']) == 1:
```

Comment:
you can just avoid the `if` with ` module.fail_json(msg="Package name(s) cannot be empty or whitespace-only")`